### PR TITLE
EN6-49 Delete Content Model

### PR DESCRIPTION
**Pull request by jeffgo10**
PR #9 needs to be merged and approved before reviewing/approving this PR. This is on deleting content model items.

### CODE REVIEW

File: src/ui/content-model/ContentModelListItem.js [hunk 1]
```diff
@@ -1,34 +1,58 @@
...
-          <MenuItem
-            className="ContentModelList__menu-item-delete"
-          >
-            <FormattedMessage id="cms.label.delete" defaultMessage="Delete" />
-          </MenuItem>
-        </DropdownKebab>
+  onClickDelete() {
+    const { onDelete, id, descr } = this.props;
+    onDelete({ id, descr });
+  }
```
[the remark this answers is not in the document]
Comment (maitojepoy) [reply src/ui/content-model/ContentModelListItem.js]: done @nicpuddu. just changed parameter to `this.props`. thought `onDelete` is still required to deconstruct. 

I strictly don't put inline functions in rendered event calls as function will recreate those every render (as also what Giuseppe recommends). But if you insist to put the inline function call, let me know and I'll adjust.

**Comment by maitojepoy** [comment]
changes pushed @nicpuddu. please review

**Comment by jeffgo10** [comment]
sorry my bad. I used my other account to write comments. @maitojepoy is my other github account 😆

File: src/state/modal/__tests__/selectors.test.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+describe('state/modal/selectors', () => {
+  it('getModal(state) returns the modal object', () => {
+    const selected = getModal(TEST_STATE);
+    expect(selected).toEqual(TEST_STATE.modal);
+  });
+
+  it('verify getVisibleModal selector', () => {
+    const selected = getVisibleModal(TEST_STATE);
+    expect(selected).toEqual('modalId');
```
[the remark this answers is not in the document]
Comment (jeffgo10) [reply src/state/modal/__tests__/selectors.test.js]: done @nicpuddu

File: src/state/modal/selectors.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const getModal = state => state.modal;
+export const getVisibleModal = state => state.modal.visibleModal;
+export const getInfo = state => state.modal.info;
```
[the remark this answers is not in the document]
Comment (jeffgo10) [reply src/state/modal/selectors.js]: changed to `createSelector` on both `getVisibleModal` and `getInfo`

**Pull request merged**